### ISSUE TITLE
Tracking - Update CAIP-10/19 for new max length & charset

### ISSUE THREAD

**Issue by bumblefudge** [ChainAgnostic/caip-js#30]

**Comment by eli-lim** [comment]
Is this repo still maintained? 🤔

**Comment by bumblefudge** [comment]
Yes, passively.  This is a tracking issue to warn users of the javascript library that it is out of date with recent slight changes to the spec.  PRs welcome if you'd like to close this issue by updating the string validation rules!
